indexer/indexer_tests/categories_test.cpp

```diff
@@ -404,6 +404,7 @@ UNIT_TEST(CategoriesIndex_AllCategories)
 
   index.AddAllCategoriesInAllLangs();
   // Consider deprecating this method if this bound rises as high as a million.
+  LOG(LINFO, ("Number of nodes in the CategoriesIndex trie:", index.GetNumTrieNodes()));
```

Comment:
Do we need this log in tests?

Reply:
Yep, it's quite informative, and needed to ensure that the bound we selected below is good enough.

Reply:
Ok